cruise.umple/src/generators/Generator_CodePlainRequirementsDoc.ump

```diff
@@ -43,10 +43,34 @@ class PlainRequirementsDocGenerator
     ReqTreeSet = new TreeSet<>(defaultComparator);
     // Add all requirements found in the model to the TreeSet 
     ReqTreeSet.addAll(model.getAllRequirements().values());
-    // Creating paragraph elements from the TreeSet
-    for (Requirement x : ReqTreeSet) {
-      String text = Requirement.translateToHTML(x.getStatement(), x.getLanguage());
-      doc.createParagraphElement("" + x.getIdentifier() + ": " + text, false);
+    // Creating paragraph elements from the TreeSet below
```

Comment:
This whole set of changes is VASTLY more complex than needed. 

Please completely rewrite